extension.json

```diff
@@ -43,7 +43,7 @@
 	},
 
 	"ResourceModules": {
-		"ext.pageApprovals.modules": {
+		"ext.pageApprovals.resources": {
```

Comment:
I missed this in the bigger PR, but typically this name does not include "modules" or "resources", since everything defined here is a resource.

Reply:
Yes. Its just a naming convention that most MW extensions follow. Its either .modules or .resources.

Reply:
Which extension include that? None of our recent extensions do and I also haven't seen other extensions doing that. They usually end with just the name, or ".styles" or ".scripts" if the module contains only the one kind of resource.

Reply:
You can call it `FooBar` and it will work. Name format does not matter